frontend/src/App.css

```diff
@@ -0,0 +1,6 @@
+.navbar{
+    background-color: #020301;
+}
+#nav-link{
```

Comment:
Can you put this css inside of its own file named Navbar.css. That way we know where the styles for the navbar live. Also when you create the css file put it inside of a new folder names styles. You will have to create the styles folder inside of the src folder.